### PR TITLE
[ZH] Add `Help centre/Website`

### PR DESCRIPTION
**Pull request by FlyingFish23**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

中午好~

### CODE REVIEW

File: wiki/Help_centre/Website/zh.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+**如果你个人介绍里的内容根据我们的[社区准则](/wiki/Rules)被判定为不适宜展示，则相关内容会被移除。**
+
+在这种情况下，你可以发邮件给 [accounts@ppy.sh](mailto:accounts@ppy.sh)，并与支持团队进一步讨论。
+
+在任何情况下，如果你不确定内容是不是适合放进个人介绍里，则你随时可以发私信给 [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) 的任何一位成员，他们会解答你的问题。
+
+### 我需要用电子邮箱地址来注册吗？ {#email-purpose}
+
+**是的。我们需要你的电子邮箱地址才能把密码重置请求和需要使用的验证码发送给你。**
```
Comment (Sunnyboy971) [inline wiki/Help_centre/Website/zh.md:91]:
```suggestion
**是的。我们需要你的电子邮箱地址才能按需把密码重置请求和验证码发送给你。**
```

这里调一下语序比较通顺

File: wiki/Help_centre/Website/zh.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  - site
+  - block
+  - forum
+  - ticket
+  - appearance
+  - message
+  - page
+  - profile
+  - missing
+  - stats
```
Comment (Sunnyboy971) [inline wiki/Help_centre/Website/zh.md:12]:
```suggestion
  - stats
  - 网站
  - 屏蔽
  - 论坛
  - 支持请求
  - 消息
  - 个人档案
  - 页面
  - 统计数据
  - 外观
```
翻译标签，便于搜索查阅

**Review by Sunnyboy971 — CHANGES_REQUESTED** [review]

File: wiki/Help_centre/Website/zh.md (hunk 2 above)
Comment (FlyingFish23) [reply wiki/Help_centre/Website/zh.md:12]: 我一直以为这个不用翻译呢！！谢谢大佬指点！！

File: wiki/Help_centre/Website/zh.md (hunk 1 above)
Comment (FlyingFish23) [reply wiki/Help_centre/Website/zh.md:91]: 👌🏻

**Review by Sunnyboy971 — APPROVED** [review]

**Pull request merged**